source/game/kart/KartParamFileManager.cc

```diff
@@ -77,8 +76,7 @@ EGG::RamStream KartParamFileManager::getVehicleStream(Vehicle vehicle) const {
     auto *file = reinterpret_cast<ParamFile<KartParam::Stats> *>(m_kartParam.file);
     ASSERT(file);
     void *offset = &file->params[idx];
-    u32 size = sizeof(KartParam::Stats);
-    return EGG::RamStream(reinterpret_cast<u8 *>(offset), size);
+    return EGG::RamStream(offset, sizeof(KartParam::Stats));
```

Comment:
This can be improved further by sending in `&file->params[idx]` instead of `offset`.